core-web/libs/edit-content/src/lib/components/dot-edit-content-form/dot-edit-content-form.component.html

```diff
@@ -32,11 +35,13 @@
                 [dotTabViewPrepend]="prependContent"
                 [dotTabViewAppend]="appendContent"></ng-template>
 
-            <!-- First tab is not lazy -->
+            <!-- First tab with same pattern as other tabs -->
             @if ($firstTab()) {
```

Comment:
I suggest creating a let variable for `$firstTab()` because it is used more times below.

Reply:
I did something more deeper here.